okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -173,18 +159,13 @@ protected ChannelEssentials buildEssentials() {
         throw new RuntimeException("Unknown negotiation type: " + negotiationType);
     }
 
-    ClientTransportFactory transportFactory = new OkHttpClientTransportFactory(
-        host, port, authorityHost, executor, socketFactory, connectionSpec);
-    Runnable terminationRunnable = null;
-    // We shut down the executor only if we created it.
-    if (transportExecutor == null) {
-      terminationRunnable = new Runnable() {
-        @Override
-        public void run() {
-          SharedResourceHolder.release(DEFAULT_TRANSPORT_THREAD_POOL, executor);
-        }
-      };
-    }
-    return new ChannelEssentials(transportFactory, terminationRunnable);
+    ClientTransportFactory transportFactory = new ClientTransportFactory() {
+      @Override
+      public ClientTransport newClientTransport() {
+        return new OkHttpClientTransport(
+                host, port, authorityHost, executor, socketFactory, connectionSpec);
```

Comment:
Nit: Line wrapping seems pretty inconsistent throughout this code.  2 things stick out to me:

1. Google Style says indent +4 for a wrapped line.  
2. Line continuations break at the highest semantic boundary.   Here it is done correctly, but it is different in NettyChannelBuilder line 166.  I would prefer consistency over a hard and fast line breaking rule.  Which one would prefer?  Could you make that the same for the lines that are changed?